### PR TITLE
Removing deprecated function

### PR DESCRIPTION
**Pull request by jstucke**
- removed deprecated function
- added config files for pre-commit

### CODE REVIEW

**Comment by dorpvom** [comment]
# [Codecov](https://codecov.io/gh/fkie-cad/common_helper_mongo/pull/2?src=pr&el=h1) Report
> Merging [#2](https://codecov.io/gh/fkie-cad/common_helper_mongo/pull/2?src=pr&el=desc) into [master](https://codecov.io/gh/fkie-cad/common_helper_mongo/commit/3d7db864d6f5a8e1d65e609a04f05566e83eee09&el=desc) will **not change** coverage by `%`.
> The diff coverage is `n/a`.

[Impacted file tree graph](https://codecov.io/gh/fkie-cad/common_helper_mongo/pull/2?src=pr&el=tree)

```diff
@@            Coverage Diff            @@
##            master        #2   +/-   ##
=========================================
  Coverage   100.00%   100.00%           
=========================================
  Files            6         6           
  Lines          158       149    -9     
=========================================
- Hits           158       149    -9     
```



------

[Continue to review full report at Codecov](https://codecov.io/gh/fkie-cad/common_helper_mongo/pull/2?src=pr&el=continue).
> **Legend** - [Click here to learn more](https://docs.codecov.io/docs/codecov-delta)
> `Δ = absolute <relative> (impact)`, `ø = not affected`, `? = missing data`
> Powered by [Codecov](https://codecov.io/gh/fkie-cad/common_helper_mongo/pull/2?src=pr&el=footer). Last update [3d7db86...7a7667f](https://codecov.io/gh/fkie-cad/common_helper_mongo/pull/2?src=pr&el=lastupdated). Read the [comment docs](https://docs.codecov.io/docs/pull-request-comments).

File: .isort.cfg [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[settings]
+line_length=120
+default_section = THIRDPARTY
+known_first_party=analysis,compare,helperFunctions,install,intercom,objects,plugins,
```
Comment (weidenba) [inline .isort.cfg]: We do not have all these dependencies, do we?

File: .pylintrc [hunk 2]
```diff
@@ -0,0 +1,392 @@
+[MASTER]
+
+init-hook='import sys; sys.path.append("./src")'
```
Comment (weidenba) [inline .pylintrc]: there is no src folder in this project

File: .pylintrc [hunk 3]
```diff
@@ -0,0 +1,392 @@
...
+
+# Add files or directories to the blacklist. They should be base names, not
+# paths.
+ignore=CVS
+
+# Add files or directories matching the regex patterns to the blacklist. The
+# regex matches against base names, not paths.
+ignore-patterns=test/,src/bin
```
Comment (weidenba) [inline .pylintrc]: there is no /src/bin to ignore

File: common_helper_mongo/.pylintrc [hunk 4]
```diff
@@ -0,0 +1,392 @@
+[MASTER]
```
Comment (weidenba) [inline common_helper_mongo/.pylintrc]: why do we need the pylintrc two times?

File: pytest.ini [hunk 5]
```diff
@@ -1,4 +1,9 @@
 [pytest]
-addopts = --pep8 -v
-pep8ignore = 
-    *.py E501
+addopts = --flake8 -v
+flake8-ignore =
+    *.py E501 W503
+    install.py E402
```
Comment (weidenba) [inline pytest.ini]: there is no install.py src/plugins or /bin/

**Review by weidenba — CHANGES_REQUESTED** [review]

**Review by weidenba — APPROVED** [review]

**Pull request merged**